daemon/main.go

```diff
@@ -88,6 +89,7 @@ var (
 	// autoIPv6NodeRoutes automatically adds L3 direct routing when using direct mode (-d)
 	autoIPv6NodeRoutes    bool
 	bpfRoot               string
+	clusterName           string
```

Comment:
I've added the cluster name to `clusterConfiguration` in `pkg/node`

Reply:
OK, I'll remove this and once that change is merged, I'll consume `clusterConfiguration` in the IPCache. I'll make that a TODO in the code.

Reply:
TODO added.